<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QMainWindow" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>465</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StoreManagement</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/newPrefix/product.png</normaloff>:/newPrefix/product.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 77, 113);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>55</width>
    <height>55</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelUn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>94</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:14pt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Username&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPass">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>257</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:14pt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Password&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../StoreManagement/Images-Icons/icons8-user-100.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../StoreManagement/Images-Icons/icons8-login-100.png</normaloff>../StoreManagement/Images-Icons/icons8-login-100.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="UnLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>221</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(190, 190, 142);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="PassLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(190, 190, 142);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorLabel">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>40</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../StoreManagement/qt.qrc"/>
  <include location="../StoreManagement/Images-Icons/qtt.qrc"/>
 </resources>
 <connections/>
</ui>
